docs/USAGE.md

```diff
@@ -1,20 +1,70 @@
+## Usage
+
+- [How do I interact with the app?](#interacting-with-the-app)
+- [How do I stop and start the server?](#stopping-and-starting-the-server)
+- [How do I run tests?](#running-tests)
+- [How do I rebuild Docker containers?](#rebuilding-docker-containers)
+- [How do I serve the front-end?](#serving-ember)
+- [Do I need special environment variables?](#environment)
+- [How do I push changes to GitHub?](#pushing-changes)
+
+### Interacting with the app
+
+You'll generally interact with the app using `docker-compose`. You can choose not to if you're contributing often enough to not need to worry about dependency management. But if you're short on time, Docker is probably the happier path to get up and running quickly.
+
+Here are conversions of your typical Elixir project commands:
+
+- `iex -S mix phoenix.server` → `docker-compose run web iex -S mix phoenix.server`
+- `mix run priv/repo/seeds.exs` → `docker-compose run web mix run priv/repo/seeds/.exs`
+- `mix test` → `docker-compose run test mix test`
+- and so on...
+
+These follow the basic format of `my-command with arguments` → `docker-compose run mix TARGET_CONTAINER my-command with arguments`.
```

Comment:
`mix` shouldnt be in this ->`docker-compose run mix TARGET_CONTAINER my-command with arguments`